src/pypendency/container.py

```diff
@@ -73,39 +73,15 @@ def set(self, identifier: str, service: object, tags: Optional[Set[Tag]] = None)
             for tag in tags:
                 self.__add_service_to_tag_group(tag, identifier)
 
+    def has(self, identifier: str) -> bool:
```

Comment:
has_identifier_registered or smth more meaningful?

Reply:
This function existed previously, changing it would break the API. I only moved it down because of call order